.github/workflows/validate.yml

```diff
@@ -4,20 +4,23 @@ on: [push, pull_request]
 
 jobs:
   linux:
+    strategy:
+      matrix:
+        jdk: ['11', '17']
     runs-on: 'ubuntu-latest'
     steps:
     - uses: actions/checkout@v1
     - uses: actions/setup-java@v1
       with:
-        java-version: '11'
+        java-version: {{ matrix.jdk }}
```

Comment:
Did you see that the workflow seems to have some issues still? (https://github.com/parttimenerd/jmc/actions/runs/1728551346)

Reply:
I did not. But I consider working on this when I'm able to run successfully run all tests.

Reply:
As a side note: the syntax is `${{ matrix.jdk }}` :)